test/functional/ui/float_spec.lua

```diff
@@ -1329,6 +1329,54 @@ describe('float window', function()
       end
     end)
 
+    it("would not break 'minimal' style with statuscolumn set", function()
+      command('set number')
+      command('set signcolumn=yes')
+      command('set colorcolumn=1')
+      command('set cursorline')
+      command('set foldcolumn=1')
+      command('set statuscolumn=%l%s%C')
```

Comment:
is there an existing test for "minimal" we can add this line to? that avoids an extra, verbose test case.